R/Connection.R

```diff
@@ -146,28 +146,36 @@ setGeneric(
 
 #' @rdname odbcConnectionColumns
 #' @param column_name The name of the column to return, the default returns all columns.
+#' @param exact If TRUE the intention is to match the identifier arguments exactly,
+#' rather than allow pattern-based-matching / wildcards.  Note, this is implemented
+#' only for select back-ends / not supported across the board.
 #' @export
 setMethod(
   "odbcConnectionColumns",
   c("OdbcConnection", "Id"),
-  function(conn, name, column_name = NULL) {
+  function(conn, name, column_name = NULL, exact = FALSE) {
```

Comment:
Do we want to make this an argument to the generic, rather than the method? Also I think we should probably tweak the generic docs to make it clear that `name` is actually a pattern.

Reply:
Thanks / good call.
Done.